src/cpu.h

```diff
@@ -25,19 +25,20 @@ typedef enum flag {
 	Z_FLAG = 7,
 } Flag;
 
-typedef union Register {
+typedef union DoubleWordReg {
 	struct {
 		uint8_t high; // A, B, D, H
 		uint8_t low; // F, C, E, L
 	} bytes;
 	uint16_t word;
```

Comment:
I think this should be called `dword` then, shouldn't it? 🤔

Reply:
Yes. `high` and `low` could be renamed to `hi` and `lo` if thats good with you